lib/database_connection.rb

```diff
@@ -0,0 +1,16 @@
+require 'pg'
+
+class DatabaseConnection
+
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+  
+  def self.connection
+    @connection
+  end
+
+  def self.query(sql)
```

Comment:
self.setup creates a database connection. 

Then self.connection is just a getter method for that connection, so that the connection is maintained

self.query runs SQL statements against that connection database